### PR TITLE
Various Fixes

### PR DESCRIPTION
**Pull request by SgtBears**
You need to compile the dll for this to work. This changes a few more tags, makes more tags optional, adds translations for Dynamic Names and Fixes a few text problems with the new buildings.

Included fix for URV text malfunction.

### CODE REVIEW

File: Assets/XML/Civilizations/CIV4CivilizationInfos.xml [hunk 1]
```diff
@@ -3017,15 +3017,15 @@
...
+			<ShortDescription>TXT_KEY_CIV_CONGO_SHORT_DESC</ShortDescription>
+			<Adjective>TXT_KEY_CIV_CONGO_ADJECTIVE</Adjective>
+			<Civilopedia>TXT_KEY_CIV_CONGO_PEDIA</Civilopedia>
 			<DefaultPlayerColor>PLAYERCOLOR_KHAKI</DefaultPlayerColor>
-			<ArtDefineTag>ART_DEF_CIVILIZATION_KONGO</ArtDefineTag>
+			<ArtDefineTag>ART_DEF_CIVILIZATION_CONGO</ArtDefineTag>
 			<ArtStyleType>ARTSTYLE_AFRICAN</ArtStyleType>
-			<UnitArtStyleType>UNIT_ARTSTYLE_MIDDLE_EAST</UnitArtStyleType>
+			<UnitArtStyleType>UNIT_ARTSTYLE_MIDDLE_EASTERN</UnitArtStyleType>
 			<Identifier>KON</Identifier>
```
Comment (dguenms) [inline Assets/XML/Civilizations/CIV4CivilizationInfos.xml]: Shouldn't the identifier also change?

File: Assets/XML/Civilizations/CIV4UnitArtStyleTypeInfos.xml [hunk 2]
```diff
@@ -20,17 +20,22 @@
 				<StyleUnit>
 					<UnitType>UNIT_CARAVEL</UnitType>
 					<UnitMeshGroup>
-						<EarlyArtDefineTag>ART_DEF_UNIT_ASIAN_WAR_GALLEY</EarlyArtDefineTag>
-						<MiddleArtDefineTag>ART_DEF_UNIT_ASIAN_WAR_GALLEY</MiddleArtDefineTag>
-						<LateArtDefineTag>ART_DEF_UNIT_ASIAN_WAR_GALLEY</LateArtDefineTag>
+						<EarlyArtDefineTag>ART_DEF_UNIT_ASIAN_ASIAN</EarlyArtDefineTag>
```
Comment (dguenms) [inline Assets/XML/Civilizations/CIV4UnitArtStyleTypeInfos.xml]: That looks like a mistake?

**Comment by SgtBears** [comment]
I wasn't sure how the identifier was used so I didn't want to mess with it. If you tell me where it is used I can change it.

Fixed the Asian caravel.

File: Assets/XML/Text/DynamicNames/DynamicNames.xml [hunk 3]
```diff
@@ -40,6 +40,26 @@
 	<TEXT>
 		<Tag>TXT_KEY_KINGDOM_OF</Tag>
 		<English>Kingdom of %s1</English>
+		<French>
+			<Text>Royaume du %s1</Text>
+			<Gender>Male</Gender>
+			<Plural>0</Plural>
+		</French>
+		<German>
+			<Text>K&#246;nigreich %s1:K&#246;nigreich %s1:K&#246;nigreich %s1</Text>
```
Comment (dguenms) [inline Assets/XML/Text/DynamicNames/DynamicNames.xml:49]: You don't have to escape unicode characters in XML, please just write them out in plain text to make the files more legible.

**Comment by SgtBears** [comment]
Fixed the Identifier.

Fixed the Asian caravel.

I just copied the translations from the original RFC by comparing the english. I don't know what the characters in the translations are supposed to be. I had a quick look but there doesn't look like there is a quick way of changed the unicode characters to normal characters.

**Comment by dguenms** [comment]
Can you take out all files where Github cannot display the diff and put them in a separate pull request? I'd like to see what exactly has changed.

**Comment by SgtBears** [comment]
I haven't removed the files from this merge but I added the files to a different merge so you can view them.

**Comment by dguenms** [comment]
Okay, nothing to comment on for those files.

Otherwise things would have become really confusing though. You're making it really hard for me by submitting those huge pull requests. It's much easier to separate them by purpose. That you have to put various in the title should be the first warning sign.

**At the very least** make sure that all file diffs are viewable on Github. But I'd prefer more separate pull requests because then I can merge them incrementally depending on their status instead of this all or nothing deal.

This pull request also changes a lot of unit tags. Have you made sure that Varietas Delectat still works with this? I'll hold off on this until that is the case.

**Comment by SgtBears** [comment]
Okay, I'll try to create smaller pull requests.

The updated varietas Delectat that I haven't uploaded yet is compatible. Normal varietas delectat isn't.

**Comment by SgtBears** [comment]
I uploaded the updated varietas delectat however the changelog is impossible to read.

**Comment by dguenms** [comment]
Okay, I'm waiting for the pull request against VD then.

**Comment by SgtBears** [comment]
Its in the Varietas delectat section.

**Comment by SgtBears** [comment]
Updated and have added the new leaders civilopedia texts.

**Pull request merged**